docs/api/dashboard-import/import.asciidoc

```diff
@@ -0,0 +1,95 @@
+<<dashboard-import-api-export, Export Dashboard API>> as the request body to
+this import API.
+
+==== Response body
+
+The response body will have a top level `objects` property that contains an
+array of the saved objects that were created. The order of these objects is not
+guaranteed.
+
+==== Examples
+
+The following example imports saved objects associated with and including the
+dashboard with id `942dcef0-b2cd-11e8-ad8e-85441f0c2e5c`.
+
+[source,js]
+--------------------------------------------------
+POST api/kibana/dashboards/import?exclude=index-pattern
+{
+  "objects": [
+    {
+      "id": "80b956f0-b2cd-11e8-ad8e-85441f0c2e5c",
+      "type": "visualization",
+      "updated_at": "2018-09-07T18:40:33.247Z",
+      "version": 1,
+      "attributes": {
+        "title": "Count Example",
+        "visState": "{\"title\":\"Count Example\",\"type\":\"metric\",\"params\":{\"addTooltip\":true,\"addLegend\":false,\"type\":\"metric\",\"metric\":{\"percentageMode\":false,\"useRanges\":false,\"colorSchema\":\"Green to Red\",\"metricColorMode\":\"None\",\"colorsRange\":[{\"from\":0,\"to\":10000}],\"labels\":{\"show\":true},\"invertColors\":false,\"style\":{\"bgFill\":\"#000\",\"bgColor\":false,\"labelColor\":false,\"subText\":\"\",\"fontSize\":60}}},\"aggs\":[{\"id\":\"1\",\"enabled\":true,\"type\":\"count\",\"schema\":\"metric\",\"params\":{}}]}",
+        "uiStateJSON": "{}",
+        "description": "",
+        "version": 1,
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": "{\"index\":\"90943e30-9a47-11e8-b64d-95841ca0b247\",\"query\":{\"query\":\"\",\"language\":\"lucene\"},\"filter\":[]}"
+        }
+      }
+    },
+    {
+      "id": "90943e30-9a47-11e8-b64d-95841ca0b247",
+      "type": "index-pattern",
+      "updated_at": "2018-09-07T18:39:47.683Z",
+      "version": 1,
+      "attributes": {
+        "title": "kibana_sample_data_logs",
+        "timeFieldName": "timestamp",
+        "fields": "<truncated for example>",
+        "fieldFormatMap": "{\"hour_of_day\":{}}"
+      }
+    },
+    {
+      "id": "942dcef0-b2cd-11e8-ad8e-85441f0c2e5c",
+      "type": "dashboard",
+      "updated_at": "2018-09-07T18:41:05.887Z",
+      "version": 1,
+      "attributes": {
+        "title": "Example Dashboard",
+        "hits": 0,
+        "description": "",
+        "panelsJSON": "[{\"gridData\":{\"w\":24,\"h\":15,\"x\":0,\"y\":0,\"i\":\"1\"},\"version\":\"7.0.0-alpha1\",\"panelIndex\":\"1\",\"type\":\"visualization\",\"id\":\"80b956f0-b2cd-11e8-ad8e-85441f0c2e5c\",\"embeddableConfig\":{}}]",
+        "optionsJSON": "{\"darkTheme\":false,\"useMargins\":true,\"hidePanelTitles\":false}",
+        "version": 1,
+        "timeRestore": false,
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": "{\"query\":{\"query\":\"\",\"language\":\"lucene\"},\"filter\":[]}"
+        }
+      }
+    }
+  ]
+}
+--------------------------------------------------
+// KIBANA
+
+A successful call returns a response code of `200`.
```

Comment:
This API returns 200 regardless if there are errors within the bulk operations or not. That indicates that the bulk operation itself was successful, but the user will still need to check the results for any errors that may occurred.

Reply:
Ah, good catch.